pkg/controller/transfer/dv_test.go

```diff
@@ -291,8 +291,6 @@ func dvTransferRunning() *cdiv1.ObjectTransfer {
 func dvTransferRunningWithSucceededDV() *cdiv1.ObjectTransfer {
 	t := dvTransfer(cdiv1.ObjectTransferRunning)
 	dv := createPopulatedDV()
-	dv.Kind = "DataVolume"
-	dv.APIVersion = "cdi.kubevirt.io/v1beta1"
```

Comment:
what happens otherwise?

Reply:
Test fails seems like a change related to controller runtime.  You actually added those last time we bumped libraries

Reply:
no I just copied a function that had those https://github.com/kubevirt/containerized-data-importer/pull/3078/commits/d3f39cbd313cbf47c3e899e1cfa59571debac198
But anyway I think this is the reasoning https://github.com/kubernetes-sigs/controller-runtime/pull/2633